company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise8Test.java

```diff
@@ -137,7 +145,9 @@ public void sortedOrders()
     @Test
     public void whoOrderedSaucers()
     {
-        MutableList<Customer> customersWithSaucers = null;
+        Predicate<Order> orderPredicate = order -> order.getLineItems().anySatisfy(lineItem -> "saucer".equals(lineItem.getName()));
+        Predicate<Customer> customersOrderedSaucers = customer -> customer.getOrders().anySatisfy(orderPredicate);
+        MutableList<Customer> customersWithSaucers = this.company.getCustomers().select(customersOrderedSaucers);
```

Comment:
Requesting the same formatting change here as well.

Reply:
Done